airbyte-workers/src/main/java/io/airbyte/workers/WorkerUtils.java

```diff
@@ -134,28 +134,15 @@ static void gentleCloseWithHeartbeat(final Process process,
     }
   }
 
-  @VisibleForTesting
-  static void forceShutdown(final Process process, final Duration lastChanceDuration) {
```

Comment:
This method was basically a copy of `closeProcess()`, but without the logic to actually forcibly shutdown the process (despite the name). So I opted to just remove it and use `closeProcess` in its place instead.